homeassistant/components/meteo_france/__init__.py

```diff
@@ -1,121 +1,102 @@
 """Support for Meteo-France weather data."""
+import asyncio
 import datetime
 import logging
 
 from meteofrance.client import meteofranceClient, meteofranceError
 from vigilancemeteo import VigilanceMeteoError, VigilanceMeteoFranceProxy
 import voluptuous as vol
 
-from homeassistant.const import CONF_MONITORED_CONDITIONS
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
 import homeassistant.helpers.config_validation as cv
-from homeassistant.helpers.discovery import load_platform
+from homeassistant.helpers.typing import ConfigType, HomeAssistantType
 from homeassistant.util import Throttle
 
-from .const import CONF_CITY, DATA_METEO_FRANCE, DOMAIN, SENSOR_TYPES
+from .const import CONF_CITY, DOMAIN, PLATFORMS
 
 _LOGGER = logging.getLogger(__name__)
 
 SCAN_INTERVAL = datetime.timedelta(minutes=5)
 
 
-def has_all_unique_cities(value):
```

Comment:
Should we keep this validator so that the user can validate the config?

Reply:
I don't think it's useful.

The user can still add "75001" or "Paris 1er arrondissement", it will be detected as unique but indeed, it's the same city and district.

The config flow handles this case